charts/tidb-backup/templates/scripts/_start_backup.sh.tpl

```diff
@@ -15,8 +15,8 @@ fi
 gc_life_time=`/usr/bin/mysql -h${host} -P4000 -u${TIDB_USER} ${password_str} -Nse "select variable_value from mysql.tidb where variable_name='tikv_gc_life_time';"`
 echo "Old TiKV GC life time is ${gc_life_time}"
 
-echo "Increase TiKV GC life time to 3h"
-/usr/bin/mysql -h${host} -P4000 -u${TIDB_USER} ${password_str} -Nse "update mysql.tidb set variable_value='3h' where variable_name='tikv_gc_life_time';"
+echo "Increase TiKV GC life time to {{ .Values.tikvGCLifeTime }}"
```

Comment:
```suggestion
echo "Increase TiKV GC life time to {{ .Values.tikvGCLifeTime | default 720 }}"
```

Reply:
When adding new variable to chart, always remember to set a default in case of users not providing the variable.

Reply:
done